project-keeper/src/main/java/com/exasol/projectkeeper/validators/changesfile/ChangesFileSection.java

```diff
@@ -11,7 +14,7 @@
  * </p>
  */
 public final class ChangesFileSection {
-
+    private static final Pattern FIXED_ISSUE_LINE_REGEXP = Pattern.compile("[-*]\\s*#(\\d+)\\s*:?\\s*(.+)?");
```

Comment:
For improved readability, we could use simple spaces and just replace them before compiling the regular expression: 
```suggestion
    private static final Pattern FIXED_ISSUE_LINE_REGEXP = Pattern.compile("[-*] #(\\d+) :? (.+)?".replace(" ", "\\s*"));
```

Reply:
Great idea!